editor/state.js

```diff
@@ -7,6 +7,11 @@ import refx from 'refx';
 import { reduce, keyBy, first, last, omit, without, flowRight } from 'lodash';
 
 /**
+ * WordPress dependencies
+ */
+import { getBlockTypes } from 'blocks';
```

Comment:
Could you clarify this? Imports from 'blocks' are WordPress dependencies in other places.